nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -363,14 +358,21 @@ in
           HostKey ${k.path}
         '')}
 
-        # Allow DSA client keys for now. (These were deprecated
-        # in OpenSSH 7.0.)
-        PubkeyAcceptedKeyTypes +ssh-dss
```

Comment:
This should be mentioned in the release notes. Otherwise people might get locked out.

Reply:
Yep, this is important to mention in the release notes. I don't think it's been fully removed yet, just deprecated. 

> OpenSSH 7.0 and greater similarly disable the ssh-dss (DSA) public key algorithm. It too is weak and we recommend against its use. It can be re-enabled using the HostKeyAlgorithms configuration option:

https://www.openssh.com/legacy.html

Reply:
can be found here: `nixos/doc/manual/release-notes/rl-1803.xml`

Reply:
```diff
diff --git a/nixos/doc/manual/release-notes/rl-1803.xml b/nixos/doc/manual/release-notes/rl-1803.xml
index c1fe692cee..8bc9cd9133 100644
--- a/nixos/doc/manual/release-notes/rl-1803.xml
+++ b/nixos/doc/manual/release-notes/rl-1803.xml
@@ -72,6 +72,16 @@ following incompatible changes:</para>
       <option>services.pgmanage</option>.
     </para>
   </listitem>
+  <listitem>
+    <para>
+      The OpenSSH service no longer enables support for DSA keys by default,
+      which could cause a system lock out. DSA support was
+      <link xlink:href="https://www.openssh.com/legacy.html">deprecated in OpenSSH 7.0</link>,
+      due to it being too weak. To re-enable support, add
+      <literal>PubkeyAcceptedKeyTypes +ssh-dss</literal> to the end of your
+      <option>services.openssh.extraConfig</option>.
+    </para>
+  </listitem>
 </itemizedlist>
 
 </section>
```

Look reasonable?